src/Upload/StreamableUploader.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may ob`tain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Upload;
+
+use GuzzleHttp\Psr7\BufferStream;
+use GuzzleHttp\Psr7\Request;
+
+class StreamableUploader extends ResumableUploader
+{
+    const DEFAULT_WRITE_CHUNK_SIZE = 262144;
+
+    public function __construct(...$args)
```

Comment:
Yeah, it's not necessary, but I was trying to avoid redeclaring all of the constructor arguments. Alternatively, I could add a hook at the end of the AbstractUploader's __construct() to use.